src/results/generator.jl

```diff
@@ -31,17 +31,22 @@
 `Generator` results keys:
 - `size_kw` Optimal generator capacity
 - `lifecycle_fixed_om_cost_after_tax` Lifecycle fixed operations and maintenance cost in present value, after tax
-- `year_one_fixed_om_cost__before_tax` fixed operations and maintenance cost over the first year, before considering tax benefits
+- `year_one_fixed_om_cost_before_tax` fixed operations and maintenance cost over the first year, before considering tax benefits
 - `lifecycle_variable_om_cost_after_tax` Lifecycle variable operations and maintenance cost in present value, after tax
 - `year_one_variable_om_cost_before_tax` variable operations and maintenance cost over the first year, before considering tax benefits
 - `lifecycle_fuel_cost_after_tax` Lifecycle fuel cost in present value, after tax
 - `year_one_fuel_cost_before_tax` Fuel cost over the first year, before considering tax benefits
-- `average_annual_fuel_used_gal` Gallons of fuel used in each year
-- `year_one_to_battery_series_kw` Vector of power sent to battery in year one
-- `year_one_to_grid_series_kw` Vector of power sent to grid in year one
-- `year_one_to_load_series_kw` Vector of power sent to load in year one
-- `year_one_energy_produced_kwh` Total energy produced in year one
-- `average_annual_energy_produced_kwh` Average annual energy produced over analysis period
+- `annual_fuel_consumption_gal` Gallons of fuel used in each year
+- `production_to_battery_series_kw` Vector of power sent to battery in year one
+- `production_to_grid_series_kw` Vector of power sent to grid in year one
+- `production_to_load_series_kw` Vector of power sent to load in year one
+- `year_one_energy_produced_kwh` Total energy produced in year one # TODO: Remove?
```

Comment:
@Bill-Becker Do you think we should remove the output `year_one_energy_produced_kwh` for Generator and Wind? These outputs are multiplied by "levelization factor", but "levelization factor" is always 1.0 for these techs, so `year_one_energy_produced_kwh` will always equal `annual_energy_produced_kwh`.

Reply:
Yeah I think we remove it. We're generally getting rid of the "year_one" production outputs, right?

Reply:
I think we want to keep the `year_one_energy_produced_kwh` output for `PV` because PV actually does have a degradation input, and therefore year_one will be different than `annual_energy_produced_kwh`. Although, we might need to apply 1 year of degradation b/c it's technically "year_zero" (I think)? 

Regardless, I removed `year_one_energy_produced_kwh` from `Generator` and `Wind` outputs in https://github.com/NREL/REopt.jl/pull/160/commits/c9fba873219143312b50b9e717b8103326f34a29

Reply:
Agreed, and I'm good with sweeping that year_zero/year_one production under the rug for now.